### DISCUSSION TITLE
program run and figma design does not look the same

### DISCUSSION THREAD

**Discussion by expeditionersjoklumarkar** [ParthJadhav/Tkinter-Designer#414] [Q&A]
![image](https://github.com/ParthJadhav/Tkinter-Designer/assets/157220151/6284553f-9419-44e6-aa36-6fcd68ef2178)
![image](https://github.com/ParthJadhav/Tkinter-Designer/assets/157220151/9af4b0a6-aeb3-457f-bdd6-1add65b4cc19)

hello everyone. i am trying to make a design in figma for our project. i tried to create the front page first and the attached pictures looks like that :'( it does not look like the design i made on the site and when exported by tkinterdesigner... i do not know the exact keys on the internet so i was not able to see the parts where I made a mistake.. can someone help? thanks guys

ps. i also noticed that they did not copy the font design i have chosen :'(.. srry for the questions i am just a beginner... thanks

**Comment by jishnupl07** [comment]
Hello, Jishnu here. 
Let's address the issues one by one. 
Fonts:  all the fonts in Figma are not supported in Tkinter. 
You can further Google the supported TKinter fonts.  
BG Image: When I was a beginner, I even experienced the same issue. 
This is because you have resized the image (in Figma) to show only the required contents in the frame. But probably, the image is bigger than the frame itself, and you are only viewing what you want. 
but when you try creating the GUI, the whole image (size>frame) will be put as the respective asset, with the white patches on the areas hidden in Figma. You view the white patches as the Tkinter designer uses `anchor = 'nw'`. 

Solution : 
Crop the bg image to fit in the frame (in Figma) 
use CTRL+mouseLEFT to drag the bg image's edges and crop it just right to fit in the frame. 
then you'll get your requirements satisfied!
